### PR TITLE
Add the /ctp command

### LINKED ISSUE #293: [Feature Request] /ctp only at spectator mode
So, you know that spectator mode can tp to the player. Now, it will be cool if you can tp to specific coordinate with spectator mode. **I don't know if this possible or not**. If not possible, maybe just add the feature without coordinate (only tp to player)

### PR DESCRIPTION
**Pull request by xpple**
Closes #293.

Not only the UUIDs suggested by the client, but also any valid UUID of an entity will be valid as an argument to the command. Because UUIDs alone are quite meaningless, entity selectors are supported as well. For example, use `/ctp @e[type=minecraft:zombie, sort=furthest, limit=1]` to teleport to the furthest zombie known to the client. Sadly, one can't teleport to arbitrary coordinates.

### CODE REVIEW

File: src/main/java/net/earthcomputer/clientcommands/command/CTeleportCommand.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    private static final MinecraftClient client = MinecraftClient.getInstance();
+
+    public static void register(CommandDispatcher<ServerCommandSource> dispatcher) {
+        addClientSideCommand("ctp");
+
+        dispatcher.register(literal("ctp")
+                .then(argument("uuid", uuid())
+                        .suggests((context, builder) -> suggestMatching(client.world.getPlayers().stream().map(Entity::getUuidAsString), builder))
+                        .suggests((context, builder) -> suggestMatching(StreamSupport.stream(client.world.getEntities().spliterator(), false).map(Entity::getUuidAsString), builder))
+                        .executes(ctx -> teleport(ctx.getSource(), getUuid(ctx, "uuid"))))
```
Comment (Earthcomputer) [inline src/main/java/net/earthcomputer/clientcommands/command/CTeleportCommand.java]: As of the latest version of clientcommands, the `uuid` arguemnt is unnecessary, as we fixed `EntityArgumentType` to allow uuids

**Review by Earthcomputer — CHANGES_REQUESTED** [review]
Looks good other than this one thing

File: src/main/java/net/earthcomputer/clientcommands/command/CTeleportCommand.java (hunk 1 above)
Comment (xpple) [reply src/main/java/net/earthcomputer/clientcommands/command/CTeleportCommand.java]: What fix exactly?

Comment (Earthcomputer) [reply src/main/java/net/earthcomputer/clientcommands/command/CTeleportCommand.java]: https://github.com/Earthcomputer/clientcommands/commit/e29ecfc447fa07d62927005d4fd2da931b5a4a81
https://github.com/Earthcomputer/clientcommands/commit/e12288977aaae874e12b42774f6822dbf072255e

**Review by Earthcomputer — APPROVED** [review]

**Pull request merged**